docs/visual/faq.md

```diff
@@ -0,0 +1,143 @@
+
+### How do I show lazy-loaded content in my screenshot?
+You need to trigger the lazy-loaded content so that it loads and displays in your web page. How to do this depends on how the lazy-loaded content is triggered to load.
+
+For example, if content is loaded based on scrolling down to a particular section of your web page, then you will need to scroll to that section using JavaScript:
+
+```js
+window.scroll(0, 1000);
+```
+
+### Why is my screenshot not full-page?
+Screener captures a full-page screenshot by default in all browsers. But the height of the full-page screenshot is based on the scrollable height of the main document body.
+
+If the body is not scrollable, then the screenshot captured will appear to be cut off. For example, if the "overflow" style on the body is set to "hidden", then the body cannot be scrolled.
+
+#### Are there workarounds?
+Yes, but the workaround depends on how the website CSS was built and understanding what is causing the body to not be scrollable.
+
+If the overflow style on the body is "hidden", then the client could execute the following JS in an Execute Script command in the test to set body overflow to `'auto'` before capturing a snapshot:
+
+```js
+document.body.style.overflow = 'auto'
+```
+
+
+## Visual E2E
+
+### What Sauce Labs browsers are supported by Visual E2E?
+Visual E2E is unified with Sauce VMs, and uses Sauce browsers exclusively. For a detailed list, see [Visual E2E Supported Browsers and Operation Systems](https://docs.saucelabs.com/visual/e2e-testing/supported-browsers).
+
+### How do I test smaller mobile web viewport sizes for responsive design testing?
+This can be done by setting [`viewportSize` in your `sauce:visual` capabilities](/visual/e2e-testing/commands-options/#saucevisual-capability-options).
+
+### How do I run functional tests only after integrating Visual E2E?
+If you want to run their functional tests only, you do not need to modify any of your visual integration code. Just change your WebDriver endpoint from `hub.screener.io` to `ondemand.saucelabs.com`.
+
+### Does Screener capture `iframes`?
+Yes; this is a Visual E2E feature. Look for [`iframes`](/visual/e2e-testing/commands-options) and [`iframesOptions`](/visual/e2e-testing/commands-options).
+
+
+
+## Component Testing
+
+### What Storybook versions are supported?
+screener-storybook supports Storybook version 3.0 to 6.x (latest).
+
+### Why is Screener not picking up data from my CI build?
+For screener-storybook and screener-runner tests run within a CI tool, Screener will automatically set build, branch, and commit options if you are using one of the following CI tools:
+
+* Jenkins
+* CircleCI
+* Travis CI
+* Visual Studio Team Services
+* Codeship
+* GitLab CI
+* Drone
+* Bitbucket Pipelines
+* Semaphore
+* Buildkite
+
+This works by pulling data from environment variables in the CI job.
+<details><summary><strong>Click here to view sample code containing expected Environment variables</strong></summary>
```

Comment:
This "click here" how does it supposed to work?

Reply:
@luishernandezv , this is what that feature looks like:

https://user-images.githubusercontent.com/56411016/139954260-9d526cb0-9bd6-4189-87c6-0d2dbf351124.mov

Reply:
Oh cool, whatever you prefer. I think the splitting by CI Tool is nicer but up to you

Reply:
Hi @luishernandezv , I've split it by CI tool into tabs. Can you please take a look at the revision?

Reply:
Nice!